### DISCUSSION TITLE
HA e integrazione con termostati honeywell

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#816] [HUB: Home Assistant]
Buongiorno, 

il mio idraulico sta per farmi un preventivo per dei termostati wifi della honeywell, ancora non conosco il modello ma credo si tratti del T3R.
Qualcuno di voi sa se questi sistemi sono interfacciabili con HA al fine di gestire delle automazioni? 
Grazie.

**Comment by MarioBomb** [comment]
Io uso il sistema Evohome della Honeywell ed è interfacciato con HA, ho dei dubbi sul T3R.
